data/ui/style_box.blp

```diff
@@ -19,16 +19,6 @@ template StyleBox : Box {
     hexpand: true;
     }
 
-  Button copy_button {
-    tooltip-text: _("Copy Style");
-    styles ["flat"]
-    Image {
-      hexpand: false;
-      icon-name: "edit-copy-symbolic";
-      pixel-size: 20;
-    }
-  }
-
   Button delete_button {
```

Comment:
Perhaps not in scope either, but maybe we should switch the delete and edit button for consistency